src/test/java/com/salesforce/dynamodbv2/PutTest.java

```diff
@@ -149,4 +157,26 @@ void putOverwriteHkRkTable(TestArgument testArgument) {
         });
     }
 
+    @ParameterizedTest(name = "{arguments}")
+    @ArgumentsSource(DefaultArgumentProvider.class)
+    void putAttributeNotExists(TestArgument testArgument) {
+        testArgument.forEachOrgContext(org -> {
+            PutItemRequest putItemRequest = new PutItemRequest()
+                .withTableName(TABLE1)
+                .withItem(ItemBuilder.builder(testArgument.getHashKeyAttrType(),
+                    HASH_KEY_VALUE_NEW)
+                    .someField(S, SOME_FIELD_VALUE_NEW)
+                    .build())
+                .withConditionExpression("attribute_not_exists(#hk)")
+                .withExpressionAttributeNames(ImmutableMap.of("#hk", HASH_KEY_FIELD));
```

Comment:
...also this string?

Reply:
fixed in 87f4f54e01482a4b7c2b6d1c7a9b51ed25870e1c